gradle.properties

```diff
@@ -1,4 +1,3 @@
 org.gradle.caching=true
 org.gradle.configureOnDemand=true
 org.gradle.parallel=true
-org.gradle.workers.max=4
```

Comment:
Now that `-Xmx` in `options.forkOptions.jvmArgs` actually applies to the compiler, I hope this will no longer be needed locally